performance/src/lib/t-test.ts

```diff
@@ -0,0 +1,15 @@
+import ttest = require("ttest");
```

Comment:
Please rename this file `ttest.ts`, otherwise this is called by our jest configuration 🙃